spec/models/group/verein_spec.rb

```diff
@@ -6,19 +6,43 @@
 require 'spec_helper'
 
 describe Group::Verein do
+  let(:hidden) { described_class.hidden }
 
   it '.hidden creates hidden verein in root group without children' do
-    hidden = described_class.hidden
     expect(hidden).to be_deleted
     expect(hidden.reload).to have(0).children
   end
 
-  it '.hidden may have additional deleted children ' do
-    hidden = described_class.hidden
+  it '.hidden may have additional deleted children' do
```

Comment:
```suggestion
  it 'hidden Vereine may have additional deleted children' do
```
try to avoid being to technical when describing test cases